pkg/apis/pipeline/v1/artifact_types.go

```diff
@@ -0,0 +1,18 @@
+package v1
+
+// Artifact represents an artifact within a system, potentially containing multiple values
+// associated with it.
+type Artifact struct {
+	Name   string          `json:"name,omitempty"`   // The artifact's identifying category name
+	Values []ArtifactValue `json:"values,omitempty"` // A collection of values related to the artifact
+}
+
+// ArtifactValue represents a specific value or data element within an Artifact.
+type ArtifactValue struct {
+	Digest map[string]string `json:"digest,omitempty"` // Algorithm-specific digests for verifying the content (e.g., SHA256)
```

Comment:
Lets set this to something like:

```
map[Algorithm]string
```

And we can alias it like:
```
type Algorithm string
```

This is so that we can limit the choices of `Algorithm` to things like `sha256, sha1` etc. in the future. e.g. Limit them to https://github.com/in-toto/attestation/blob/main/spec/v1/digest_set.md#supported-algorithms